scripts/check-build-prerequisites.sh

```diff
@@ -74,6 +74,19 @@ else
     wrong_version
 fi
 
+# Rustup
+app="rustup"
+app_min="1.25.2"
+check_which $app $app_min
+app_v=$(${app} --version 2>/dev/null)
+echo -e "Checking rustup version... \c"
+if [ $(echo $app_v | grep -c -E "^1\.(2(5\.[2-9]\d*|[6-9]\d*\.\d*)|[3-9]\d+\.\d+)") -eq 1 ]
```

Comment:
```suggestion
if [ $(echo $app_v | grep -c -E "^1\.(2(5\.[2-9]\d*|[6-9]\.\d*)|[3-9]\d+\.\d+)") -eq 1 ]
```

Reply:
Here's a railroad diagram of the original (i.e: my) version of the expression (courtesy of https://regexper.com):

![image](https://github.com/aws/aws-cdk/assets/411689/9aafe4b8-5a1b-4735-bb91-a264c6a3d610)

Reply:
cool site. Based on this it seems romains expression works. Can you confirm elena?